do not use workaround on aarch64 as luajit does not need it now
from:
https://github.com/rspamd/rspamd/issues/3563

--- a/contrib/lua-lpeg/lptypes.h
+++ b/contrib/lua-lpeg/lptypes.h
@@ -154,7 +154,7 @@
 
 /* Special workaround for luajit lightuserdata limitations with GC64 */
 #if defined(WITH_LUAJIT) && INTPTR_MAX == INT64_MAX && \
-	!(defined(_X86_) || defined(__x86_64__) || defined(__i386__) || defined(__powerpc__))
+	!(defined(_X86_) || defined(__x86_64__) || defined(__i386__) || defined(__powerpc__) || defined(__aarch64__))
 # define LPEG_LUD_WORKAROUND 1
 void * lpeg_allocate_mem_low(size_t sz);
 void lpeg_free_mem_low(void *p);
